### PR TITLE
fix: contribution action

### PR DESCRIPTION
**Pull request by LashaJini**
[FX-NNNN]

### Description

- escape single quotes in PR title

### How to test

- Check PR in [davinci](https://github.com/toptal/davinci/pull/1868)
- Check the [contribution action](https://github.com/toptal/davinci/actions/runs/3984719070/jobs/6831844894), should succeed even when title contains single quote

### Development checks

- [x] Add changeset according to [guidelines](https://github.com/toptal/davinci-github-actions/blob/master/_docs/contribution/changeset-guidelines.md) (if needed)

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-anvil ping reviewers` - Ping FX team for review

</details>

### CODE REVIEW

File: .changeset/healthy-items-happen.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'davinci-github-actions': patch
+---
+
+- escape single quotes in PR's title
```
Comment (OndrejTuma) [inline .changeset/healthy-items-happen.md:5]: nit: for better changelog
```suggestion
### notify-jira-about-contribution
- escape single quotes in PR's title
```

File: notify-jira-about-contribution/action.yml [hunk 2]
```diff
@@ -66,10 +66,21 @@ runs:
...
+        echo {
+          "data": {
+            "author": "${{ env.AUTHOR }}", 
+            "author_url": "${{ env.AUTHOR_URL }}", 
+            "title": "${{ env.TITLE }}", 
+            "number": "${{ env.NUMBER }}", 
+            "url": "${{ env.URL }}",
+            "repo": "${{ env.REPO }}"
+          }
+        } | sed "s/'/\\\\'/g" | curl -H 'Content-type: application/json' --data-binary @- $JIRA_HOOK
```
Comment (OndrejTuma) [inline notify-jira-about-contribution/action.yml:83]: Have you tested it?

Comment (LashaJini) [reply notify-jira-about-contribution/action.yml:83]: Didn't have time, will do it later :bow:

**Comment by LashaJini** [comment]
@toptal-bot run package:alpha-release

File: notify-jira-about-contribution/action.yml [hunk 3]
```diff
@@ -57,19 +57,30 @@ runs:
       if: ${{ fromJson(steps.is-pr-eligible.outputs.result) == true }}
       uses: andymckay/labeler@1.0.4
       with:
-        add-labels: "contribution"
+        add-labels: 'contribution'
 
     - name: Notify JIRA
-      if: ${{ fromJson(steps.is-pr-eligible.outputs.result) == true && fromJson(steps.jira-issue-can-be-created.outputs.result) == true }}
+      # if: ${{ fromJson(steps.is-pr-eligible.outputs.result) == true && fromJson(steps.jira-issue-can-be-created.outputs.result) == true }}
```
Comment (LashaJini) [inline notify-jira-about-contribution/action.yml:63]: for testing purposes

**Review by OndrejTuma — APPROVED** [review]
I've tested the generate_data function and it works correctly 👌

**Review by dmaklygin — APPROVED** [review]

**Pull request merged**